guardrails/main.py

```
@@ -0,0 +1,118 @@
+    ]
+)
+
+# Initialize Rails configurations
+try:
+    logger.info("Loading guardrails configuration from ./rails...")
+    config = RailsConfig.from_path("./rails")
+    rails = LLMRails(config)
+    
+    logger.info("Guardrails configurations loaded successfully")
+except Exception as e:
+    logger.error(f"Failed to load guardrails configurations: {e}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)